apis/v0/jwt.go

```diff
@@ -30,6 +33,9 @@ const (
 	jwtJustificationsKey string = "justs"
 )
 
+// marshalFn takes an input and return a byte array representation of the input.
+type marshalFn func(in interface{}) ([]byte, error)
```

Comment:
nits: use `any` instead of `interface{}`
Since 1.18, Go uses any as the alias for interface{}.